lib/core/presentation/pages/ai/ai_page.dart

```diff
@@ -47,9 +52,29 @@ class AIPageState extends State<AIPage> {
 
   final TextEditingController _textController = TextEditingController();
 
+  late StreamSubscription<bool> keyboardSubscription;
+
   @override
   void initState() {
     super.initState();
+
+    var keyboardVisibilityController = KeyboardVisibilityController();
```

Comment:
please define this outside initState, same level of keyboardSubscription

Reply:
not related to keyboard, but I think AIChatMessage should be define in domain layer instead here